aspnetcore/toc.md

```diff
@@ -73,31 +73,29 @@
 
 # [Fundamentals](fundamentals/index.md)
 ## [Application Startup](fundamentals/startup.md)
+## [Dependency Injection (Services)](fundamentals/dependency-injection.md)
 ## [Middleware](fundamentals/middleware.md)
-## [Working with Static Files](fundamentals/static-files.md)
-## [Routing](fundamentals/routing.md)
-## [URL Rewriting Middleware](fundamentals/url-rewriting.md)
```

Comment:
@Rick-Anderson Did you mean to drop this one? It fell out of the TOC on this PR.

Reply:
No - thanks for the heads up, I'll get that fixed ASAP